modules/swagger-codegen/src/main/resources/Java/libraries/jersey2/ApiClient.mustache

```diff
@@ -725,6 +726,11 @@ public class ApiClient {
         buildResponseHeaders(response),
         respBody);
     }
+    } finally {
+      try {
+        response.close();
+      } catch (Exception e) {}
```

Comment:
Can be. In general I usually handle the subject of resources in that way, so I avoid even the exception Java NullPointerException.
Could be:
    } finally {
      try {
        if (response != null) response.close();      
      } catch (Exception e) {
        throw new RuntimeException(e);
      }
    }

What do you think ?